src/JwtCredentialManager.js

```diff
@@ -0,0 +1,84 @@
+/* @flow */
+import type { ICredentialsManager } from './CredentialManager';
+
+type Seconds = number;
+
+export type JwtCredentials = {
+  exp: Seconds,
```

Comment:
maybe we can just inline `number`

Reply:
i wanted to use the fancy Opaque types but all tooling still throws up on it